cypress/integration/todo.spec.ts

```diff
@@ -0,0 +1,61 @@
+const newToDo1 = 'New Todo 1';
+const newToDo2 = 'New Todo 2';
+const existingToDo1 = 'Open Todo list example';
+const existingToDo2 = 'Check the other examples';
+const existingToDo3 = 'Use Angular ngRx Material Starter in your project';
+
+describe('TODO Actions', () => {
+  beforeEach(() => {
+    cy.visit('/#/examples/todos');
+  });
+
+  it('should be able to add a new todo with enter or + button click', () => {
+    cy.get('input[placeholder="I am going to..."]').type(newToDo1 + '{enter}');
+    cy.get('body').should('contain', newToDo1);
+
+    cy.get('input[placeholder="I am going to..."]').type(newToDo2);
+    cy.get('[arialabel="add todo"]').click();
+    cy.get('body').should('contain', newToDo2);
```

Comment:
What about giving the todo list a `data-testid` attribute, and check if the list item exists in the lists instead of in the body itself?

Reply:
items have data-testid already, using that now, thx